docs/learn/dapp-developer/read-profile-data.md

```diff
@@ -7,115 +7,106 @@ description: Learn how to read profile data from your Universal Profile.
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
-# Read Universal Profile Data
+# Read Data from your Universal Profile
 
-<div style={{textAlign: 'center', color: 'grey'}}>
-  <img
-    src={require('/img/learn/up_view.png').default}
-    alt="Universal Profile example on universalprofile.cloud"
-    width="800"
-  />
-<br/>
-<i>A <a href="https://wallet.universalprofile.cloud/0x6979474Ecb890a8EFE37daB2b9b66b32127237f7?network=testnet">Universal Profile</a> as seen on UniversalProfile.cloud</i>
-<br /><br />
-</div>
-
-:::info
-
-The full code of this example can be found in the 👾 [lukso-playground](https://github.com/lukso-network/lukso-playground/tree/main/universal-profile) repository.
-
-:::
-
-:::tip Universal Profile Explorer
-
-The explorer on 🔮 [universalprofile.cloud](https://universalprofile.cloud/) indexes all deployed Universal Profiles on the LUKSO network. You can try out the following examples with any Universal Profile address.
-
-:::
-
-## Setup
-
-To easily interact with a profile you should use the ⚒️ [erc725.js](https://npmjs.com/package/@erc725/erc725.js) library. It is able to fetch and encode profile and contract data easily. You can install it in your project using:
+## Install [erc725.js](https://npmjs.com/package/@erc725/erc725.js) library
```

Comment:
There is also content here:

https://docs.lukso.tech/tools/erc725js/getting-started

Let's make sure we don't duplicate content from that side also